App/Sagas/TextileSagas.js

```diff
@@ -141,8 +143,19 @@ export function * stopNode () {
 
 export function * getPhotoHashes ({thread}) {
   try {
-    const photoData = yield call(IPFS.getPhotos, null, -1, thread)
-    yield put(IpfsNodeActions.getPhotoHashesSuccess(thread, photoData))
+    const hashes = yield call(IPFS.getPhotos, null, -1, thread)
+    let data = []
+    for (const hash of hashes) {
+      try {
+        const meta = yield call(IPFS.getHashData, hash, '/comment')
```

Comment:
just shoves the comments in with hashes right on first get. after all the rest works we can combine this in go

Reply:
yup nice